cvat/apps/engine/media_extractors.py

```diff
@@ -602,10 +600,22 @@ def __iter__(self):
                         return
 
 class IChunkWriter(ABC):
+    IMAGE_EXT = 'jpeg'
+    POINT_CLOUD_EXT = 'pcd'
```

Comment:
The `Mpeg4ChunkWriter` class also inherits from the `IChunkWriter`, I don't think that it's the correct place for these attributes. You can place it and `_write_pcd_file` to `ZipChunkWriter` and inherit `ZipCompressedChunkWriter` from `ZipChunkWriter`